ext/File-Glob/t/basic.t

```diff
@@ -1,9 +1,9 @@
 #!./perl
 
+our %Config;
 BEGIN {
     chdir 't' if -d 't';
     @INC = '../lib';
-    use Config;
```

Comment:
you do not want to remove `use Config` but replace it with what was there before: `require Config; Config->import`